### ISSUE TITLE
Editing a proposal again

### ISSUE THREAD

**Issue by Nagalim** [Daology/Support#12]
URL: http://daology.org/u/nagalim/proposals/c62b183b7e0ed133fbd0a948488ee57aae5a8693/edit 

 USER AGENT: Mozilla/5.0 (Linux; Android 5.0.1; SCH-I545 Build/LRX22C) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.105 Mobile Safari/537.36

please detail your issue below

---

This is still a big issue.  Basically, if i edit anything on my phone and then edit it on the computer i can never again edit it with my phone because it stores a cached version.  Clearing cache doesn't help.  If i have to clear all data for chrome i simply won't use daology on my phone because that's stupidly inconvenient.  Anyway, maybe i just shouldn't be using daology on my phone, but that's a real shame.  It would seriously stunt my adoption of daology for motions and articles and stuff.

**Comment by CoinGame** [comment]
Yeah I agree and we're really sorry about that. This isn't something that
we're trying to make a big public statement about but we're in the process
of laying out daology v2.0, which is going to be a major rebuild. In the
coming weeks we're going to take a look at the current issues and try to
solve the ones we can. The v2 platform will be taking the development focus
though and it should be easier to handle these issues. I would say hold off
on editing from mobile for now and we'll see if we can push this into the
fix list before we focus on v2.

**Comment by Nagalim** [comment]
Ok, so should i make you aware of all the current problems that you might put the fixes in the 2.0 update, or will most of it all be so different anyway that i should wait on bug reporting till after the update?

**Comment by CoinGame** [comment]
The front end is going to get a major rewrite so i think it's better to hold off. There will at the very least be different issues to make :)
